### PR TITLE
Add an informational comment on why we've got mini_racer in the Gemfile

### PR DESCRIPTION
**Pull request by bagedevimo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -97,6 +97,12 @@ gem 'sass'
 gem 'sass-rails'
 gem 'coffee-rails'
 gem 'uglifier', '>=1.0.3'
+# Provide a JS runtime to execjs without needing to
+# have node, bin, or similar installed on the relevant
```
Comment (tom93) [inline Gemfile:101]: Should "bin" be "bun"?

Comment (bagedevimo) [reply Gemfile:101]: hilariously, no: https://bun.sh/

Comment (tom93) [reply Gemfile:101]: So what's "bin"?

Comment (bagedevimo) [reply Gemfile:101]: oh, shit, i misread your comment, and my comment. fml.

**Pull request merged**